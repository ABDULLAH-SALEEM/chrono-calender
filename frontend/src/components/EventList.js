import React from "react";
const EventList = () => <div>Event List Component</div>;
export default EventList;
