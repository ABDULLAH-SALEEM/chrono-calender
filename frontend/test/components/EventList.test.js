import React from "react";
import { render, screen } from "@testing-library/react";
import EventList from "../../src/components/EventList";

describe("EventList", () => {
  it("renders without crashing", () => {
    render(<EventList />);
    expect(screen.getByText(/Event List Component/i)).toBeInTheDocument();
  });

  it("matches snapshot", () => {
    const { asFragment } = render(<EventList />);
    expect(asFragment()).toMatchSnapshot();
  });
});
