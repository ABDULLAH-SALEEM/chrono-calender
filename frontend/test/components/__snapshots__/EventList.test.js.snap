// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventList matches snapshot 1`] = `
<DocumentFragment>
  <div>
    Event List Component
  </div>
</DocumentFragment>
`;
